Error when creating a type with only 1 param.

Example:

```
blogcli tx blog create-user manolete
```

returns 

```
panic: runtime error: index out of range [1] with length 1

goroutine 1 [running]:
github.com/jgimeno/blog/x/blog/client/cli.GetCmdCreateUser.func1(0xc0010d4dc0, 0xc0010de390, 0x1, 0x1, 0x0, 0x0)
        /Users/jgimeno/tmp/blog/x/blog/client/cli/txUser.go:23 +0x5ef
github.com/spf13/cobra.(*Command).execute(0xc0010d4dc0, 0xc0010de370, 0x1, 0x1, 0xc0010d4dc0, 0xc0010de370)
        /Users/jgimeno/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:842 +0x47c
github.com/spf13/cobra.(*Command).ExecuteC(0xc0010aa000, 0x44b23e5, 0xc0010aa000, 0x4ceebc1)
        /Users/jgimeno/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:950 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
        /Users/jgimeno/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:887
github.com/tendermint/tendermint/libs/cli.Executor.Execute(0xc0010aa000, 0x4eef9f0, 0x2, 0xc00109a620)
        /Users/jgimeno/go/pkg/mod/github.com/tendermint/tendermint@v0.33.6/libs/cli/setup.go:89 +0x3c
main.main()
        /Users/jgimeno/tmp/blog/cmd/blogcli/main.go:72 +0x34e
```

Probably it expects always the email param directly without safe checks.

--- a/changelog.md
+++ b/changelog.md
@@ -10,6 +10,7 @@
 ### Fixes:
 
 * Running `starport add type...` multiple times no longer breaks the app
+* Running `appcli tx app create-x` now checks for all required args. -#173.
 
 ## `v0.0.10-rc.3`
 

--- a/starport/templates/typed/templates/x/{{appName}}/client/cli/tx{{TypeName}}.go.plush
+++ b/starport/templates/typed/templates/x/{{appName}}/client/cli/tx{{TypeName}}.go.plush
@@ -17,7 +17,7 @@ func GetCmdCreate<%= title(TypeName) %>(cdc *codec.Codec) *cobra.Command {
 	return &cobra.Command{
 		Use:   "create-<%= TypeName %><%= for (i, field) in Fields { %> [<%= field.Name %>]<% } %>",
 		Short: "Creates a new <%= TypeName %>",
-		Args:  cobra.MinimumNArgs(1),
+		Args:  cobra.MinimumNArgs(2),
 		RunE: func(cmd *cobra.Command, args []string) error {
       <%= for (i, field) in Fields { %>args<%= title(field.Name) %><%= if (field.Datatype != "string") {%>, _<%}%> := <%= if (field.Datatype == "string") {%>string<%} else {%>strconv.Parse<%= title(field.Datatype) %><%}%>(args[<%= i %>])
       <% } %>

